deploy/chart/templates/0000_50_olm_08-catalog-operator.deployment.yaml

```diff
@@ -62,6 +62,11 @@ spec:
             httpGet:
               path: /healthz
               port: {{ .Values.catalog.service.internalPort }}
+          env:
+          {{ if and .Values.installType (eq .Values.installType "ocp") }}
```

Comment:
@ecordell have you ever found a way to handle white-space chomping with back-to-back if statements like this? I must have tried every sensible permutation in these situations 🙁